test/data_5bus.jl

```diff
@@ -0,0 +1,540 @@
+        name="demand_d",
+        id=3,
+        available=true,
+        power_systems_type="PowerLoad",
+        region=[z2],
+        #peak_load=peak_load,
+    )
+
+    ####################
+    ##### Transmission #####
+    #####################
+
+    line = ACTransportTechnology{ACBranch}(
+        name="test_branch",
+        start_region=z1,
+        end_region=z2,
+        existing_line_capacity=100,
+        maximum_new_capacity=900,
+        line_loss=0.05,
+        capital_cost=LinearCurve(5000.0),
+        available=true,
+        power_systems_type="TransportTechnology",
+        id=1,
+        base_power=1.0,
+        financial_data=tech_financials,
+    )
+
+    ts_line_2024 = SingleTimeSeries(;
+        data=TimeArray(tstamp_2024_ops, ts_wind_2024_data),
+        name="ops_variable_cap_factor",
+        scaling_factor_multiplier=get_initial_capacity,
+    )
+    ts_line_2028 = SingleTimeSeries(;
+        data=TimeArray(tstamp_2028_ops, ts_wind_2028_data),
+        name="ops_variable_cap_factor",
+        scaling_factor_multiplier=get_initial_capacity,
+    )
+
+    finances = PortfolioFinancialData(2025, 0.07, 0.05, 0.03)
+
+    #####################
+    ##### Portfolio #####
+    #####################
+
+    p_5bus = Portfolio(finances)
+
+    PSIP.set_name!(p_5bus, "test")
+    #PSIP.add_financials!(p_5bus, finances)
+    #PSIP.add_financials!(p_5bus, tech_financials)
+    PSIP.set_financial_data!(p_5bus, finances)
+    PSIP.add_region!(p_5bus, z1)
+    PSIP.add_region!(p_5bus, z2)
+    PSIP.add_technology!(p_5bus, t_th)
+    PSIP.add_technology!(p_5bus, t_wind)
+    PSIP.add_technology!(p_5bus, t_pv1)
+    PSIP.add_technology!(p_5bus, t_pv2)
+    PSIP.add_technology!(p_5bus, t_th_exp)
+    PSIP.add_technology!(p_5bus, t_demand_b)
+    PSIP.add_technology!(p_5bus, t_demand_c)
+    PSIP.add_technology!(p_5bus, t_demand_d)
+    PSIP.add_technology!(p_5bus, t_stor)
+    PSIP.add_technology!(p_5bus, line)
+
+    PSIP.add_time_series!(p_5bus, t_th, ts_th_cheap_inv_capex)
+    PSIP.add_time_series!(p_5bus, t_th_exp, ts_th_exp_inv_capex)
+    IS.add_time_series!(p_5bus.data, t_th, ts_thermal_2024; year="2024", rep_day=1)
+    IS.add_time_series!(p_5bus.data, t_th, ts_thermal_2028; year="2028", rep_day=2)
+    IS.add_time_series!(p_5bus.data, t_th_exp, ts_thermal_2024; year="2024", rep_day=1)
+    IS.add_time_series!(p_5bus.data, t_th_exp, ts_thermal_2028; year="2028", rep_day=2)
+
+    IS.add_time_series!(p_5bus.data, t_wind, ts_wind_2024; year="2024", rep_day=1)
+    IS.add_time_series!(p_5bus.data, t_wind, ts_wind_2028; year="2028", rep_day=2)
+    PSIP.add_time_series!(p_5bus, t_wind, ts_wind_inv_capex)
+
+    IS.add_time_series!(p_5bus.data, t_pv1, ts_pv1_2024; year="2024", rep_day=1)
+    IS.add_time_series!(p_5bus.data, t_pv1, ts_pv1_2028; year="2028", rep_day=2)
+    PSIP.add_time_series!(p_5bus, t_pv1, ts_pv1_inv_capex)
+    IS.add_time_series!(p_5bus.data, t_pv2, ts_pv2_2024; year="2024", rep_day=1)
```

Comment:
We need to have a method for add time series to a portfolio using kwargs that basically is a wrapper of this IS method.